include/oneapi/tbb/detail/_task_handle.h

```diff
@@ -34,7 +34,7 @@ class task_handle;
 
 class task_handle_task : public d1::task {
     std::uint64_t m_version_and_traits{};
-    d1::wait_context& m_wait_ctx;
+    d1::wait_tree_vertex_interface* m_wait_tree_node;
```

Comment:
May be it makes sense to use the term vertex here as well:
```suggestion
    d1::wait_tree_vertex_interface* m_wait_tree_vertex;
```